internal/rbd/clone.go

```diff
@@ -135,6 +135,9 @@ func (rv *rbdVolume) generateTempClone() *rbdVolume {
 	tempClone.ClusterID = rv.ClusterID
 	tempClone.Monitors = rv.Monitors
 	tempClone.Pool = rv.Pool
+	if rv.RadosNamespace != "" {
```

Comment:
Well, I think for the namespaced images it is not bad to have a limited user test too. The secret is reseted to any other test anyway.

Reply:
we already have tests for namespace https://github.com/ceph/ceph-csi/blob/master/e2e/rbd.go#L961-L1052 this can be tweaked to use normal user instead of created same  tests with restricted user

Reply:
fixed, plus rebased, but build are timing out now